logging/log4j1_2/appenderSrc/test/resources/log4j.xml

```diff
@@ -26,11 +26,23 @@
     <appender name="test" class="com.microsoft.applicationinsights.log4j.v1_2.ApplicationInsightsAppender">
         <param name="instrumentationKey" value="c9341531-05ac-4d8c-972e-36e97601d5ff" />
         <layout class="org.apache.log4j.PatternLayout">
-            <param name="ConversionPattern" value="%-5p %c{1} - %m%n"/>
+            <param name="ConversionPattern" value="%-5p %c - %m%n"/>
         </layout>
     </appender>
+    <appender name="console" class="org.apache.log4j.ConsoleAppender">
```

Comment:
Added this for better debugging. I was able to figure out which logger to turn off after enabling this appender.